src/pages/public_registration/PublicSummaryContent.tsx

```diff
@@ -35,6 +36,22 @@ export const PublicSummaryContent = ({ registration }: PublicRegistrationContent
               {entityDescription.abstract}
             </Typography>
           )}
+          {entityDescription.alternativeAbstracts.und && (
+            <>
+              <Accordion elevation={0}>
+                <AccordionSummary expandIcon={<ExpandMoreIcon color="primary" />} sx={{ padding: '0' }}>
+                  <Typography variant="h3" color="primary">
+                    {t('registration.description.alternative_abstract')}
+                  </Typography>
+                </AccordionSummary>
+                <AccordionDetails sx={{ padding: 0, marginBottom: '2rem' }}>
```

Comment:
er det en grunn til at det er bottom margin her?
```suggestion
                <AccordionDetails sx={{ padding: 0 }}>
```